.github/workflows/ci.yml

```diff
@@ -0,0 +1,68 @@
+on: [push, pull_request]
+
+name: Continuous integration
+
+jobs:
+  check:
+    name: Check
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions-rs/toolchain@v1
+        with:
+          toolchain: stable
+          override: true
+      - uses: actions-rs/cargo@v1
+        with:
+          command: check
+
+  test:
+    name: Test Suite
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions-rs/toolchain@v1
+        with:
+          toolchain: stable
+          override: true
+      - name: Install dependencies
+        run: sudo apt install libudev-dev
```

Comment:
Why do we need libudev?

Reply:
One of the dependencies https://crates.io/crates/udev requires it to be installed.

Reply:
https://github.com/stephanh/espresso/runs/518902830?check_suite_focus=true#step:4:70

Reply:
Ah, you're right, I missed that! It's a dependency of the `serialport` dev dependency.

You can get rid of the udev requirement by adding `default_features = false` to the `serialport` entry in `Cargo.toml`. It should still build fine.

Reply:
Done.